themes/bootstrap/static/js/color-console-output.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+let html = document.querySelector('section').innerHTML;
+
+// Find the ANSI escape sequences for coloring the text
+// const ansiPatterns = [/\^\[\[\d*m/g, /\^\[\[\d*;\d*m/g, /\^\[\[\d*;\d*;\d*m/g];
+// for (let pattern of ansiPatterns) {
+//   console.log(new Set(html.match(pattern)));  // Used in ansiSequencesToSpans
+// }
+
+// Manually format the relevant patterns
+const ansiSequencesToSpans = {
+  // '\\^\\[\\[0m': '<span>',
```

Comment:
Remove commented code

Reply:
This will be resolved by a general ANSI code parser (see below).